### PR TITLE
Add batch preview

### LINKED ISSUE #3711: Expenditures: As a user I can display batch payment preview
[Provided by Adrian via Contractors project management platform]

## Description
User has to be able to display a batch payment preview after clicking on "View all" button. Batch preview is a modal containing data table with columns: recipient id, currency and amount of funds. User can scroll through data and close modal by clicking on exit icon.

## Acceptance Criteria

- an option to display modal
- modal contains data table with: recipient id, currency, amount of funds
- an option to scroll through data if there is too many rows to display
- an option to display exit icon and close modal by clicking on it.

## Design

- [Figma Link - Batch Payment Preview](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=3807%3A291681)

reject)

### PR DESCRIPTION
**Pull request by agnieszkajarosikloj**
## Description

This PR adds modal with payment preview. After clicking 'View all' button modal with payment details is displayed.

Screenshot 2022-08-16 at 15 01 07

[Figma Link - Batch Payment Preview](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=3807%3A291681)

Resolves  #3711

### CODE REVIEW

File: src/modules/dashboard/components/ExpenditurePage/Batch/PreviewDialog/PreviewDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+              <div
+                className={classNames(styles.row, {
+                  [styles.borderTop]: index === 0,
+                })}
+              >
+                <div className={styles.left}>
+                  <MaskedAddress address={recipient} />
+                </div>
+                <div className={styles.middle}>{token}</div>
+                <div className={styles.right}>{value}</div>
```
Comment (willm30) [inline src/modules/dashboard/components/ExpenditurePage/Batch/PreviewDialog/PreviewDialog.tsx:88]: I tried this with a CSV with column headings 'Recipients, Token, Amount' and the amount isn't showing in the preview. I believe "value" here should read "amount".

**Review by willm30 — CHANGES_REQUESTED** [review]
Currently, the amount values aren't showing up for me, using the following CSV data.

```csv

Recipient | Token | Amount
0xCd96D435128415F3265c32f9eaeFaCc6Be53b7D7 | xDai | 1000
0xCd96D435128415F3265c32f9eaeFaCc6Be53b7D7 | xDai | 1001
0xCd96D435128415F3265c32f9eaeFaCc6Be53b7D7 | xDai | 1002
0xCd96D435128415F3265c32f9eaeFaCc6Be53b7D7 | xDai | 1003
0xCd96D435128415F3265c32f9eaeFaCc6Be53b7D7 | xDai | 1004
0xCd96D435128415F3265c32f9eaeFaCc6Be53b7D7 | xDai | 1005
0xCd96D435128415F3265c32f9eaeFaCc6Be53b7D7 | xDai | 1006
0xCd96D435128415F3265c32f9eaeFaCc6Be53b7D7 | xDai | 1007
0xCd96D435128415F3265c32f9eaeFaCc6Be53b7D7 | xDai | 1008
```

**Review by Julianahlk — COMMENTED** [review]
I am having a similar  issue as Will, here is what my .cvs file looks like when I upload it:
Screen Shot 2022-08-18 at 4 14 45 PM

**Comment by willm30** [comment]
@agnieszkajarosikloj Tried again with the valid and invalid data and it's looking good. I did notice though that the scrollbar when present isn't styled according to the design (i.e. rounded).

**Review by Julianahlk — APPROVED** [review]
This is working well for me now 👍
Screen Shot 2022-08-29 at 11 54 39 PM

**Review by chinins — COMMENTED** [review]
The code looks good. You can update with minor things I mentioned and once I have an example of csv file from the other PR and test it I'll approve shortly.

**Comment by willm30** [comment]
> @agnieszkajarosikloj Tried again with the valid and invalid data and it's looking good. I did notice though that the scrollbar when present isn't styled according to the design (i.e. rounded).

@agnieszkajarosikloj Can you let me know what you've agreed with product re the scroll bar? If you're not planning to add it I can approve as the preview functionality is working nicely for me.

File: src/modules/dashboard/components/ExpenditurePage/Batch/PreviewDialog/PreviewDialog.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  &::-webkit-scrollbar-track {
+    border-radius: var(--radius-normal);
+    background-color: var(--colony-light-blue);
+  }
+
+  &::-webkit-scrollbar-thumb {
+    width: 5px;
+    border-radius: var(--radius-normal);
+    border-right: 2px solid var(--colony-light-blue);
+    border-left: 2px solid var(--colony-light-blue);
```
Comment (willm30) [inline src/modules/dashboard/components/ExpenditurePage/Batch/PreviewDialog/PreviewDialog.css:92]: @agnieszkajarosikloj The scrollbar is looking a little warped for me. 

I tried this instead. Let me know what you think.

``` css
&::-webkit-scrollbar-thumb {
    border-radius: 6px;
    border: 2px solid var(--colony-light-blue);
    background-color: color-mod(var(--temp-grey-blue-7) alpha(65%));
  }
```

Before:
scrollbar1

After:
scrollbar2

**Review by willm30 — APPROVED** [review]
Looking good 😎

**Pull request merged**